common/remote/hooks/azure/azurefile.cpp

```diff
@@ -311,6 +312,11 @@ unsigned __int64 AzureFileIO::getStatistic(StatisticKind kind)
     return stats.getStatistic(kind);
 }
 
+IFile * AzureFileIO::queryFile() const
+{
+    return file.get();
```

Comment:
trivial/style: am surprised you use .get() in these circumstances (vs relying on operator to convert). I've always found these Owned/Linked 'get' methods a tad confusing, giving the impression they may be linking.

(and not used in git GitRepositoryFileIO change)